css/src/atomics/width.scss

```diff
@@ -35,6 +43,10 @@ $all-widths: list.join($universal-widths, $mobile-incompatible-widths);
 		width: 100% !important;
 	}
 
+	.width-fit {
```

Comment:
TL;DR - `.width-fit-content`

It would be nice to have this class but let's follow the naming conventions:

https://design.learn.microsoft.com/atomics/overview.html